src/index.js

```diff
@@ -28,6 +28,7 @@ export default function aphrodisiac(jss, options) {
 
     const style = rules.reduce(mergeStyles, {})
     sheet.addRule(className, style, {className})
+    sheet = sheet.deploy()
```

Comment:
why do you need deploy? You may be trapped by the fact that insertRule api doesn't update the devtools representation, so you don't see the rules, but they are there, to see them you can go to the `style.sheet.rules`